suricata.yaml.in

```diff
@@ -264,6 +264,7 @@ outputs:
         - ikev2
         - krb5
         - snmp
+        #- rfb
```

Comment:
I'd say we enable this by default, unless there are reasons this shouldn't be done.